pkg/organizations/client.go

```diff
@@ -426,11 +426,14 @@ func (c *Client) UpdateOrganization(ctx context.Context, opts UpdateOrganization
 		//
 		// Deprecated:  Use DomainData instead.
 		Domains []string `json:"domains,omitempty"`
+
+		// The Organization's metadata.
+		Metadata map[string]string `json:"metadata,omitempty"`
```

Comment:
I noticed we have two `UpdateOrganizationOpts` definitions, and one of them already had metadata. Are we able to remove this struct and use the existing one? Then I'm pretty sure we can just pass opts into `c.JSONEncode` directly.

Reply:
Good call! Looks like this is how we do it for updateuser too so now with this update it will follow the same pattern.